packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -907,6 +907,11 @@ mixin WidgetsBinding on BindingBase, ServicesBinding, SchedulerBinding, GestureB
   Element get renderViewElement => _renderViewElement;
   Element _renderViewElement;
 
+  bool _appHasBootStapped = false;
+
+  @override
+  bool get framesEnabled => super.framesEnabled && _appHasBootStapped;
```

Comment:
"BootStapped" isn't english... what did you mean? :-)

Reply:
maybe "rootWidgetAttached"?

Reply:
I misspelled bootstrapped. but your suggestion may be better. will update

Reply:
fwiw you ended up going with a different name -- i'm fine with either, just wanted to mention it since you said above that you like the specific name i'd suggested.

Reply:
The rootWidgetAttached will confuse with the existing https://api.flutter.dev/flutter/widgets/WidgetsBinding/isRootWidgetAttached.html. This field actually need to be set to true right before attaching the root widget. That is why i change the name.